views/message.pug

```diff
@@ -0,0 +1,20 @@
+extends layout
+
+block content
+  .container
+    if messages.success
+      div.text-success
+        for success in messages.success
+          div= success.msg
+    if messages.error
+      div.text-danger
+        for error in messages.error
+          div= error.msg
+
+    h3 Call Rest API Form
+
+    form(method='POST')
```

Comment:
There is no need to do so, the default is the own page. The controller fires the post method of the /message URL call with the form parameters payload.